psql2mysql.py

```diff
@@ -15,11 +15,13 @@
 # limitations under the License.
 #
 
-import re
-import six
 from oslo_config import cfg
 from oslo_log import log as logging
 from prettytable import PrettyTable
+
+import re
+import six
+
```

Comment:
In Python usually this order is used:

* Python imports (like `re`)
* site-package imports (like six, oslo, prettytable or alchemy)
* local imports (modules from the same code)

Reply:
my local `flake8-2.7` complained about the order, but seems like it is too old